lago_python_client/models/tax_rate.py

```diff
@@ -0,0 +1,22 @@
+from typing import Optional
+
+from pydantic import BaseModel
+
+from ..base_model import BaseResponseModel
+
+
+class TaxRate(BaseModel):
+    name: Optional[str]
+    code: Optional[str]
+    value: Optional[float]
+    description: Optional[str]
+
+
+class TaxRateResponse(BaseResponseModel):
+    lago_id: str
+    name: str
+    code: str
+    value: float
+    description: str
```

Comment:
I guess `description` is an optional field

Reply:
good spot